### ISSUE TITLE
PlayersList = new Mongo.Collection  seems to be missing

### ISSUE THREAD

**Issue by tim333** [realdavidturnbull/leaderboard#2]
The previous github thingy at: https://github.com/realdavidturnbull/leaderboard/tree/842894bb6d248a83d351a9c14173a0f61568c7fa

for the end of the Methods chapter has: 

// Create a MongoDB Collection
PlayersList = new Mongo.Collection('players');

as the first lines of the js file, but in this restructured version it seems missing, stopping the app working.

I stuck the above line in a js file in the main folder on my local computer and it seems to fix the problem.

Ta for the tutorial book by the way.

**Comment by rzZzv** [comment]
Yeah, as it's said in the docs http://docs.meteor.com/#/full/structuringyourapp - «Example File Structure» Section

                  lib/                      # common code like collections and utilities

**Comment by rzZzv** [comment]
But anyways, this is was a great tutorial for a beginner like me, thanks =)
